aspnetcore/signalr/configuration.md

```diff
@@ -278,18 +278,46 @@ Additional options for configuring timeout and keep-alive behavior are available
 
 | Option | Default value | Description |
 | ------ | ------------- | ----------- |
-| `ServerTimeout` | 30 seconds (30,000 milliseconds) | Timeout for server activity. If the server hasn't sent a message in this interval, the client considers the server disconnected and triggers the `Closed` event (`onclose` in JavaScript). This value must be large enough for a ping message to be sent from the server **and** received by the client within the timeout interval. The recommended value is a number at least double the server's `KeepAliveInterval` value to allow time for pings to arrive. |
+| `withServerTimeout` | 30 seconds (30,000 milliseconds) | Timeout for server activity in milliseconds. If the server hasn't sent a message in this interval, the client considers the server disconnected and triggers the `Closed` event (`onclose` in JavaScript). This value must be large enough for a ping message to be sent from the server **and** received by the client within the timeout interval. The recommended value is a number at least double the server's Keep-Alive interval (`withKeepAliveInterval`) value to allow time for pings to arrive. |
```

Comment:
I think the text version of `KeepAliveInterval` should not be capitalized -- keep-alive interval, like you do with server timeout.